### PR TITLE
feat(api): rewrite

### LINKED ISSUE #357: Enhance API files
Make them feature complete with 1.6.2

### PR DESCRIPTION
**Pull request by wizard-28**
## Purpose

Closes #357 

## Progress

- [x] Move to standard logging library for `fancy()` message.
- [x] Rewrite `ask()` into `confirm()`
- [x] Delete `config.py` (for now)

## Checklist

- [x] I confirm that I have read the [contributing guidelines](https://github.com/pacstall/pacstall/blob/develop/CONTRIBUTING.md), and this pull request is abiding by all the clauses stated in the guideline.

### CODE REVIEW

File: pacstall/api/prompt.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+# You should have received a copy of the GNU General Public License
+# along with Pacstall. If not, see <https://www.gnu.org/licenses/>.
+
+from os import environ
+
+from pacstall.api.color import Foreground as Fg
+from pacstall.api.color import Style as St
+
+
+def confirm(question: str, default: bool = True) -> bool:
```
Comment (saenai255) [inline pacstall/api/prompt.py:33]: Should a neutral option exist? What do you think

**Review by saenai255 — CHANGES_REQUESTED** [review]

File: pacstall/api/prompt.py (hunk 1 above)
Comment (wizard-28) [reply pacstall/api/prompt.py:33]: Nah, I asked @Henryws, he said that `confirm()` should always have a default value.

**Review by saenai255 — APPROVED** [review]

File: pacstall/__main__.py [hunk 2]
```diff
@@ -27,43 +27,50 @@
...
     if args.command in ["install", "remove", "upgrade"]:
         lock_file = open("/var/lock/pacstall.lock", "w")
         while True:
             try:
                 lockf(lock_file, LOCK_EX | LOCK_NB)
+                log.debug("Lock acquired")
                 break
             except OSError:
-                message.fancy("error", "Pacstall is already running another instance")
+                log.error("Pacstall is already running another instance")
```
Comment (D-Brox) [inline pacstall/__main__.py:66]: Shouldn't this be a warning instead, since pacstall will run when it's able to

Comment (wizard-28) [reply pacstall/__main__.py]: https://github.com/pacstall/pacstall/pull/471/commits/9b0857511ca0f2b729d0854e65f9c89c7700a7bf

**Review by D-Brox — APPROVED** [review]

**Pull request merged**